### DISCUSSION TITLE
Search for Hardware Part Numbers

### DISCUSSION THREAD

**Discussion by tbnobody** [tbnobody/OpenDTU#191] [Announcements]
To determine the exact type of inverter we are currently searching for Hardware Part Numbers with the correspondent inverter type (e.g. HM-1500, HM-1200, etc.)
This would be helpful for a better implementation of the limit setting.

For more details please see: https://github.com/tbnobody/OpenDTU/issues/35#issuecomment-1267621246

**Comment by xpertsavenue** [comment]
Ich habe einen HM1200: HW part number ist 269619201-> Müsste als ebenfalls wie schon in deiner Tabelle 10	12	10	01  sein

**Reply by ghost** [reply]
HM-600:
Bootloader Version | 0.1.4
Firmware Version | 1.0.10
Firmware Build Date | 2021-11-01 08:42:00
Hardware Part Number | 269553683
Hardware Version | 256

**Reply by stefan123t** [reply]
@tbnobody, hier ein Beispiel für die Hardware und Software Version aus der Hoymiles Dokumentation.
![image](https://user-images.githubusercontent.com/11159753/219500114-23bed3df-b28e-4912-b936-bdddb4c92eb1.png)

Die bisherige Hardware Part Number wird in der Hoymiles SMiles Cloud also etwas anders angezeigt:
Nicht als Dezimalzahl `0d269553683 = 0x10111013` sondern m.E. Hardware Ver. `H01.00.11` und Software Ver. `V01.00.13`

Hier sind es immer nur je eine Hex-Stelle/-Ziffer für die Major und Minor Version 
und zwei Hex-Stellen für die letzten beiden Stellen der Hardware / Software Version.

**Comment by tiskluth** [comment]
HM-800:
Bootloader Version | 0.1.4
Firmware Version | 1.0.10
Firmware Build Date | 2021-11-01 08:42:00
Hardware Part Number | 269565952
Hardware Version | 01.00

**Comment by die-maus** [comment]
HM-700:
Bootloader Version | 0.1.4
Firmware Version | 1.0.10
Firmware Build Date | 2021-11-01 08:42:00
Hardware Part Number | 269557783
Hardware Version | 256

**Comment by B00gieman** [comment]
HM 400
Bootloader Version	0.1.2
Firmware Version	1.0.14
Firmware Build Date	2021-12-09 12:46:00
Hardware Part Number	269500417
Hardware Version	02.00

**Comment by OutOfSync1** [comment]
HM 1500:
Bootloader Version: 0.1.0
Firmware Version: 1.0.16
Firmware Build Date: 2021-10-12 17:06:00
Hardware Part Number: 269627393
Hardware Version: 256

**Comment by oooomh** [comment]
HM-1500:
Bootloader Version | 0.1.0
Firmware Version | 1.0.18
Firmware Build Date | 2021-12-24 18:21:00
Hardware Part Number | 269627393
Hardware Version | 01.0A

BTW:  Thank you very much for this really cool piece of software including its documentation!!

**Comment by b-a-9-0** [comment]
HM-1200:
Bootloader Version	0.1.0
Firmware Version	1.0.16
Firmware Build Date	2021-10-12 17:06:00
Hardware Part Number	269619201
Hardware Version	01.00

**Comment by viktorhotel** [comment]
TSOL TSUN  M-800:
Bootloader Version | 0.1.4
Firmware Version | 1.0.8
Firmware Build Date | 2020-07-10 12:58:00
Hardware Part Number | 269553682
Hardware Version | 01.00

**Comment by eeprom23** [comment]
Model
HM-1500
Bootloader Version
Firmware Version
0.1.0
Firmware Build Date
1.0.18
2021-12-24
18:21:00
Hardware Part
Number
269627393
Hardware Version
01.10

**Comment by ce3rd** [comment]
HM-1500
Bootloader Version | 0.1.0
Firmware Version | 1.0.18
Firmware Build Date | 2021-12-24 18:21:00
Hardware Part Number | 269627393
Hardware Version | 01.0A

**Comment by gd35** [comment]
HM-350
Bootloader Version | 0.1.2
Firmware Version | 1.0.14
Firmware Build Date | 2021-12-09 12:46:00
Hardware Part Number | 269492231
Hardware Version | 512

**Comment by copemande** [comment]
HM-600
Bootloader Version | 0.1.4
Firmware Version | 1.0.8
Firmware Build Date | 2020-07-10 12:58:00
Hardware Part Number | 269553665
Hardware Version | 01.00

**Comment by madmartin** [comment]
Model | HM-600
-- | --
Bootloader Version | 0.1.4
Firmware Version | 1.0.10
Firmware Build Date | 2021-11-01 08:42:00
Hardware Part Number | 269553683
Hardware Version | 01.10




Model | HM-800
-- | --
Bootloader Version | 0.1.4
Firmware Version | 1.0.10
Firmware Build Date | 2021-11-01 08:42:00
Hardware Part Number | 269565952
Hardware Version | 01.10

**Comment by turrican944** [comment]
Model | HM-600

Bootloader Version | 0.1.4
Firmware Version | 1.0.10
Firmware Build Date | 2021-11-01 08:42:00
Hardware Part Number | 269553683
Hardware Version | 01.10

**Comment by D3R-ST3FAN** [comment]
Model	HM-1500
Bootloader Version	0.1.0
Firmware Version	1.0.18
Firmware Build Date	2021-12-24 18:21:00
Hardware Part Number	269627393
Hardware Version	01.10


Model	HM-1500
Bootloader Version	0.1.0
Firmware Version	1.0.18
Firmware Build Date	2021-12-24 18:21:00
Hardware Part Number	269627400
Hardware Version	01.00

**Comment by Alfons-Wa** [comment]
HM-700:
Bootloader Version | 0.1.4
Firmware Version | 1.0.10
Firmware Build Date | 2021-11-01 08:42:00
Hardware Part Number | 269557783
Hardware Version | 01.10

**Comment by ghost** [comment]
HM-600:

Bootloader Version | 0.1.4
Firmware Version | 1.0.10
Firmware Build Date | 2021-11-01 08:42:00
Hardware Part Number | 269553683
Hardware Version | 01.10

**Comment by DevYves89** [comment]
HM-300:
Bootloader Version | 102
Firmware Version | 10010
Firmware Build Date | 2020-07-07 14:12:00
Hardware Part Number | 269488135
Hardware Version | 02.00

**Comment by danielmiju** [comment]
<html>
<body>


Model | HM-1200
-- | --
Detected max. Power | 1200 W
Bootloader Version | 0.1.0
Firmware Version | 1.0.18
Firmware Build Date | 2021-12-24 18:21:00
Hardware Part Number | 269619201
Hardware Version | 01.00


</body>
</html>

**Comment by dirkheiden** [comment]
Model	HM-400
Detected max. Power	400 W
Bootloader Version	0.1.2
Firmware Version	1.0.14
Firmware Build Date	2021-12-09 12:46:00
Hardware Part Number	269500416
Hardware Version	02.00

**Comment by dirkheiden** [comment]
Model HM-800
Detected max. Power 800 W
Bootloader Version 0.1.4
Firmware Version 1.0.10
Firmware Build Date 2021-11-01 08:42:00
Hardware Part Number 269565952
Hardware Version 01.10

**Comment by HacksBugsAndRockAndRoll** [comment]
Model HM-700

Bootloader Version | 0.1.4
Firmware Version | 1.0.10
Firmware Build Date | 2021-11-01 08:42:00
Hardware Part Number | 269557783
Hardware Version | 256

**Comment by itschaaf** [comment]
**TSUN TSOL-M800(DE)**

Werte aus OpenDTU:
Model | HM-600
Detected max. Power | 600 W
Bootloader Version | 0.1.4
Firmware Version | 1.0.10
Firmware Build Date | 2021-11-01 08:42:00
Hardware Part Number | 269553682
Hardware Version | 01.10

**Comment by itschaaf** [comment]
**TSUN TSOL-M1600**

Werte aus OpenDTU:
Model | HM-1500
Detected max. Power | 1500 W
Bootloader Version | 0.1.0
Firmware Version | 1.0.12
Firmware Build Date | 2020-06-24 19:00:00
Hardware Part Number | 269627408
Hardware Version | 01.00

**Comment by capiman** [comment]
Modell | HM-600
Ermittelte max. Leistung | 600 W
Bootloader Version | 0.1.4
Firmware Version | 1.0.10
Firmware Erstellungsdatum | 2021-11-01 08:42:00
Hardware Teilenummer | 269553683
Hardware Version | 01.00

**Comment by ghost** [comment]
Modell | HM-1500
Ermittelte max. Leistung | 1.500 W
Bootloader Version | 0.1.0
Firmware Version | 1.0.18
Firmware Erstellungsdatum | 2021-12-24 18:21:00
Hardware Teilenummer | 269627400
Hardware Version | 01.00

**Reply by Haddicst** [reply]
Modell	HM-600
Ermittelte max. Leistung	600 W
Bootloader Version	0.1.4
Firmware Version	1.0.10
Firmware Erstellungsdatum	2021-11-01 08:42:00
Hardware Teilenummer	269553683
Hardware Version	01.10

**Comment by stefan123t** [comment]
Damit es nicht verloren geht, hier die von Hoymiles auf der Webseite aktuell gelisteten unterschiedlichen Modelle / Modellreihen mit dem dazu angegebenen Typ.

Typen | Modell
--- | ---
1-in-1 | HM-300/350/400
1-in-1 | HM-300N/350N/400N
1-in-1 | HM-300NT/350NT/400NT
1-in-1 | HMS-300/350/400/450/500
1-in-1 | HMS-350/400/450/500-NA
1-in-1 | HMS-400/450/500
2-in-1 | HM-600/600P/700/700P/800
2-in-1 | HM-600/700/800
2-in-1 | HM-600N/700N/800N
2-in-1 | HM-600NT/700NT/800NT
2-in-1 | HMS-600/700/800/900/1000
2-in-1 | HMS-700/800/900/1000-NA
2-in-1 | HMS-800-2T-LV
2-in-1 | HMS-800/900/1000
2-in-1 | MI-700LV
4-in-1 | HM-1200/1500
4-in-1 | HM-1200N/1500N
4-in-1 | HM-1200NT/1500NT
4-in-1 | HME-1200/1500
4-in-1 | HMS-1600/1800/2000
4-in-1 | HMS-1600/1800/2000 (Copy)
4-in-1 | HMS-1600/1800/2000-NA
4-in-1 | HMS-1600C/1800C/2000C
4-in-1 | HMS-1600C/1800C/2000C-NA
4-in-1 | HMS-1600D/1800D/2000D
4-in-1 | HMS-1800B/2000B
4-in-1 | MI-1200/1500
6-in-1 | HMT-1800/2250
Three-phase | HMT-1800/2250

Vermutlich muß man da ein paar auch noch korrigieren, da es wohl speziell bei den 4-in-1 Typen noch Unterschiede in der Anzahl der MPPT gibt.
